### DISCUSSION TITLE
Products Endpoint not updating products

### DISCUSSION THREAD

**Discussion by gcamvp** [Sellbrite/Sellbrite-API#26] [Q&A]
Hi, 

Hi,

I am trying to update a product already in Sellbrite via API, the product SKU is GY0229.1 and I want to update the quantity to 129.  However, I get a 200 response with data that has not been modified. The response is as if I did a GET call.

See the PUT request below:-
 

curl --location --request PUT 'https://api.sellbrite.com/v1/products/GY0229.1' \
--header 'Content-Type: application/json' \
--header 'Authorization: Basic XXXXXXXXXXXXXXXXXXXXxxxxxxxxxxxxxxxxxXXXXXXXXXXX' \
--data '{
    "sku": "GY0229.1",
    "name": "Distant Relatives (Green Colored Vinyl) - Nas & Damian \"Jr. Gong\" Marley (2LP)",
    "quantity": 129
}'

How can I make sure the product get updated via API?

Thanks in advance.

G

**Comment by gsaraiva-godaddy** [comment]
Hi,

For you to be able to update the quantity of a product you need to update his inventory through https://developer.sellbrite.com/endpoints/inventory-patch.

Please contact us again if there is any further problems.
